apps/app/src/ui/dropdownLists/tags/SelectTags.tsx

```diff
@@ -139,22 +147,35 @@ const SelectTags = ({
     });
     deleteTag(parseInt(tagId as string));
   };
+
+  const resolvedPlaceholder: string | undefined = (() => {
+    if (typeof placeholder === 'function') {
+      return placeholder(isEditionAllowed);
+    }
+    return placeholder;
+  })();
+
```

Comment:
Quel est l'intérêt de faire un IIFE ici ? 
ça alourdit le code pour une utilisé que je ne comprends pas 🤔 

```suggestion
  const computedPlaceholder = typeof placeholder === 'string' ? placeholder : placeholder?.(isEditionAllowed)

```